sources/github.js

```diff
@@ -245,3 +246,65 @@ module.exports.formatDigest = ({ issues, prs, start, end }) => {
 
   return { content, title: `Dev Digest - ${formatDates({ start, end })}` }
 }
+
+const formatAggComments = ({ enriched_comments: items }) => {
+  const type = items.length === 1 ? 'comment' : 'comments'
+  const users = Array.from(new Set(items.map(({ user }) => user.login).filter((v) => v)))
+  const links = items.map(({ html_url }) => `[${html_url.split('#issuecomment-')[1]}](${html_url})`)
+  return `${items.length} ${type}${users.size ? ` by ${users.join(', ')}` : ''} (${links.join(', ')})`
+}
+const formatAggCommits = ({ enriched_commits: items }) => {
+  const type = items.length === 1 ? 'commit' : 'commits'
+  const users = Array.from(new Set(items.map(({ author }) => author?.login).filter((v) => v)))
+  const links = items.map(({ html_url, sha }) => `[${sha.slice(0, 7)}](${html_url})`)
+  return `${items.length} ${type}${users.size ? ` by ${users.join(', ')}` : ''} (${links.join(', ')})`
+}
+const formatAggReviews = ({ enriched_reviews: items }) => {
+  const type = items.length === 1 ? 'review' : 'reviews'
+  const users = Array.from(new Set(items.map(({ user }) => user.login).filter((v) => v)))
+  const links = items.map(({ html_url }) => `[${html_url.split('#discussion_')[1]}](${html_url})`)
+  return `${items.length} ${type}${users.size ? ` by ${users.join(', ')}` : ''} (${links.join(', ')})`
+}
+
+module.exports.formatPreviously = ({ issues, prs, start, end }) => {
+  const allLinked = prs.map((pr) => pr.linked_issues).flat()
+  const all = [
+    ...issues.filter((issue) => !allLinked.includes(getID(issue))),
+    ...prs,
+  ]
+  let content = ''
+
+  if (all.length) {
+    content += `${all.length} updates${formatAggStates(all)}`
+    const grouped = all.reduce(groupByCatProj, {})
+    Object.entries(grouped).forEach(([category, byProjects]) => {
+      content += `\n\n# ${category}\n`
+      Object.entries(byProjects).forEach(([project, items]) => {
+        content += `\n## ${project}`
+        items.forEach((item) => {
+          content += `\n* ${formatItem(item)}`
```

Comment:
_minor_: why in the ss, the former has the PR creator and the latter doesn't? couldn't figure it out by scanning through the code since `formatItem`>`composeItem`>`formatUSer` all seem to be in place here... but all looks good according to #bot-cmd 👍

Reply:
incorrect screenshot. updated